wagtail/admin/tests/pages/test_create_page.py

```diff
@@ -1494,7 +1494,7 @@ def test_inline_streamfield(self):
         self.assertEqual(response.status_code, 200)
 
         # response should include HTML declarations for streamfield child blocks
-        self.assertContains(response, '<div id="sections-__prefix__-body" data-block="')
+        self.assertContains(response, '<div id="sections-__prefix__-body" data-block')
```

Comment:
```suggestion
        self.assertContains(response, '<div id="sections-__prefix__-body" data-block')
        soup = this.get_soup(response.conent)
        blockDiv = soup.find('div', {"data-controller":"w-block"})
        self.assertIsNotNone(blockDiv)
        # also check the div has an attribute for the two data values, just that they exist, not the content inside 
        
```

It would be good to bolster this test a bit more, leave the code better than we found it. 

We should test the data attributes are being used for the Stimulus controller. No need to test the JSON values, just that the attributes are used. 

Search for other tests in the codebase that use soup, see the docs here https://beautiful-soup-4.readthedocs.io/en/latest/

Reply:
```python
self.assertContains(response, '<div id="sections-__prefix__-body" data-block')
soup = self.get_soup(response.content)
blockDiv = soup.find('div', {"data-controller":"w-block"})
self.assertIsNotNone(blockDiv)
# block div should contain this attributes
self.assertTrue(blockDiv.has_attr('data-w-block-arguments-value'))
self.assertTrue(blockDiv.has_attr('data-w-block-data-value'))
```
Something like this will do the job. What do you think?

Reply:
Love it

Reply:
Hey, @lb- Can you give me few more issues to work on...

Reply:
See if you're up for tackling https://github.com/wagtail/wagtail/issues/11537 (or even part of it). If you have an idea on the approach, comment on the issue and go for it. 

There may be others but that's all I can think of now that might be a good progression from this issue. 

Another thing to do is go through unconfirmed bugs and see if any need to be reproduced, helps the community with triage but also you may find something you can give a go at fixing. Thanks for being keen on staying involved.